azure-batch/azure/__init__.py

```diff
@@ -1 +1 @@
-__path__ = __import__('pkgutil').extend_path(__path__, __name__)
\ No newline at end of file
+__import__('pkg_resources').declare_namespace(__name__)
```

Comment:
Not ok, keep the pkgutil file

Reply:
Fiexed.

Reply:
@xingwu1 You need to remove line 1, right now your file has two lines, one with pkgresources, one with pkgutil

Reply:
Fixed.